src/ui_nbgl.c

```diff
@@ -20,107 +21,97 @@ static const char* const infoContents[] = {
     VERSION
 };
 
+static nbgl_layoutSwitch_t switches[3] = {0};
```

Comment:
use a #define for the switches number (will be reused after)

Reply:
Done